src/main/kotlin/no/nav/klage/oppgave/service/KlagebehandlingMegaPutFacade.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.klage.oppgave.service
+
+import no.nav.klage.oppgave.api.view.EditerbareFelterInput
+import no.nav.klage.oppgave.domain.klage.Klagebehandling
+import no.nav.klage.oppgave.domain.kodeverk.Utfall
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+@Service
+@Transactional
+class KlagebehandlingMegaPutFacade(
+    private val klagebehandlingService: KlagebehandlingService,
+    private val vedtakService: VedtakService
+) {
+
+    fun updateEditerbareFelter(
+        klagebehandlingId: UUID,
+        input: EditerbareFelterInput,
+        innloggetIdent: String
+    ): Klagebehandling {
+        val klagebehandling =
+            klagebehandlingService.getKlagebehandlingForUpdate(klagebehandlingId, input.klagebehandlingVersjon)
+        dirtyCheckAndUpdateUtfall(input, klagebehandling, innloggetIdent)
+        //TODO: Alle de andre feltene også..
+        return klagebehandling
+    }
+
+    private fun dirtyCheckAndUpdateUtfall(
```

Comment:
Hadde ært smoothere med noe mer generisk enn en stor metode for hvert felt, men kommer ikke på noe :)